proto/serverless/instrumentation/v1/dev_mode.proto

```diff
@@ -17,10 +17,24 @@ message DevModePayload {
   // The lambda request id where this payload originated from
   string request_id = 3;
 
+  // Extracted Lambda Telemetry API data
+  optional LambdaTelemetry telemetry = 6;
+
   oneof payload {
     // The set of lambda traces that were generated via an internal extension
     TracePayload trace = 4;
     // The req or response data from the instrumented lambda function
     RequestResponse request_response = 5;
   }
 }
+
+// Lambda Telemetry API data. This data is only available for lambda functions that
+// have access to the telemetry API so it will not be included in all regions.
+message LambdaTelemetry {
+  // Init duration as reported by the metrics on the platform.initReport event
+  optional uint32 init_duration = 1;
```

Comment:
Duration is in what units, ms?

It'll be good to document that, and maybe ensure the unit is reflected in name (e.g. `init_duration_ms`)

Reply:
Yeah I like this idea 👍 I updated both the description and the variable names